CMakeLists.txt

```diff
@@ -140,7 +140,8 @@ configure_file(
 
 ################################################################
 
-if (NOT APPLE)
+if (NOT APPLE AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS "9.1")
```

Comment:
I don't think "NOT APPPLE" is needed.

Reply:
I cleaned up my commits (to only have a single commit) and removed the APPLE check.